healthcheck/healthcheck.go

```diff
@@ -53,8 +51,10 @@ type VersionInfo struct {
 // version information of the app,
 // criticalTimeout for how long to wait until an unhealthy dependent propagates its state to make this app unhealthy
 // interval in which to check health of dependencies
-// clients of type Client which contain a Checker function which is run to check the health of dependent apps
-func Create(version VersionInfo, criticalTimeout, interval time.Duration, clients []*Client) HealthCheck {
+// checkers of type Checker which are functions run to check the health of the app and/or its dependencies
+func Create(version VersionInfo, criticalTimeout, interval time.Duration, checkers ...*Checker) HealthCheck {
+
+	var clients []*Client
```

Comment:
Are these not just a list of checks now?

I think you were referring to them as statuses before, but I didn't like the idea that these should be called that.

It already it seems to make more sense by renaming clients to checkers.

Reply:
I agree, the next PR will address this. I was just worried about making too many changes all at once.